lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -169,6 +173,20 @@ SessionState::SessionState(
       static_rules_(rule_store),
       credit_map_(4, &ccHash, &ccEqual) {}
 
+/*For 5G which doesn't have response context*/
+SessionState::SessionState(
+    const std::string& imsi, const std::string& session_ctx_id,
+    const SessionConfig& cfg, StaticRuleStore& rule_store)
+    : imsi_(imsi),
+      session_id_(session_ctx_id),
+      // Request number set to 1, because request 0 is INIT call
+      request_number_(1),
+      /*current state would be CREATING and version would be 0 */
+      curr_state_(CREATING),
+      config_(cfg),
+      stored_session_version_(0),
```

Comment:
Personally, I think we can just call this `version_`, as we have a series of structs of the patterh `stored_` that is used in the storage layer.

Reply:
sure

Reply:
changed as "current_version"